docs/user_manual/keys.rst

```diff
@@ -4452,6 +4452,42 @@ minimum value for the :math:`\bar{S_{f}}` used in the
    pfset Solver.OverlandKinematic.Epsilon 1E-7           ## TCL syntax
 
    <runname>.Solver.OverlandKinematic.Epsilon = 1E-7     ## Python syntax
+  
+
+Mixed Boundary Conditions  
+
+If the top boundary is set to **OverlandKinematic** and a pfsolid file is used, this solid file can have multiple patches with mixed boundary conditions.  One patch is assigned to the overland kinematic, and up to two more patches can be assigned to seepage face cells mixed in with the overland flow.
+
+*double* **Solver.OverlandKinematic.SeepageOne** -999  
+
+This key sets the first of two available patch values for mixed boundary conditions.
+
+::
+   pfset Solver.OverlandKinematic.SeepageOne 3           ## TCL syntax
```

Comment:
Why 2 Seepage patches

We set values based on patch names in other parts of the code (e.g. BCPressure.PatchNames).

I'm a bit confused here.   "OverlandKinematic" is set for a patch.   E.G.

pfset Patch.**z-upper**.BCPressure.Type		      OverlandKinematic

So are the two other patches done with something like 

pfset Patch.**z-upper-seepage-one**.BCPressure.Type		      Seepage

pfset Patch.**z-upper-seepage-two**.BCPressure.Type		      Seepage

I'm a confused by the "mixed in" comment ,patches are non-overlapping.

Example input file might help my understanding.